erpnext/accounts/doctype/sales_invoice/sales_invoice.js

```diff
@@ -517,15 +517,6 @@ cur_frm.fields_dict.write_off_cost_center.get_query = function(doc) {
 	}
 }
 
-// project name
-//--------------------------
-cur_frm.fields_dict['project'].get_query = function(doc, cdt, cdn) {
-	return{
-		query: "erpnext.controllers.queries.get_project_name",
```

Comment:
Why remove this? maybe I don't have context here... is there any other place where query function is getting set?

Reply:
For accounting doctypes like sales invoice, `get_filtered_dimensions` is used. Since project and cost center are standard accounting dimensions, even for these two this is used in the sales invoice. Have kept the query because its still being called from Sales Order